docs/reference/core.rst

```diff
@@ -4,8 +4,8 @@
 Building Blocks
 ===============
 
-Building-block classes for configuring hybrid, asynchronous decomposition samplers,
-together with the provided :ref:`infrastructure <flow>` classes.
+Building-block classes for hybrid workflows together with the provided
+:ref:`infrastructure <flow>` classes.
```

Comment:
First core/flow/etc are just temporary and those might get merged, or at least all block will be available under `hybrid` ns, so the submodule will just be an implementation detail.

Right now I see things in `core` as primitives, like base classes subclassed by components regular (non-block-developer) users will actually use.

In other camp we have flow controlling blocks, together with samplers, decomposers and composers.

Reply:
As discussed, this will be dealt with in next iteration